DependencyInjection/Compiler/AssetsVersionCompilerPass.php

```diff
@@ -68,7 +68,7 @@ public function process(ContainerBuilder $container): void
         $runtimeDefinition->setArgument(1, $version);
 
         if (is_a($versionStrategyDefinition->getClass(), JsonManifestVersionStrategy::class, true)) {
-            $jsonManifestString = file_get_contents($version);
+            $jsonManifestString = @file_get_contents($version);
```

Comment:
i really dislike ignoring errors. could we instead do

```suggestion
            if (!file_exists($version)) {
                $this->log($container, 'The manifest file at "'.$version.'" does not yet exist');
                
                return;
            }
            $jsonManifestString = file_get_contents($version);
```

Reply:
I didn't add it in the first place because `file_get_contents` doesn't trigger an error, only a E_WARNING which doesn't halt the script. It's the cache:clear of symfony who transform this into an error afterwards.

And since the jsonManifestString is set to false, the unreadable file is handled by the next condition.

But as you wish, the log message will be clearer with your proposed solution !

Reply:
yeah, i guess the check if the contents of the file is empty would have caught it. but this seems clearer to me.

thanks for looking into the issue and providing the fix!